admin/src/admin.html

```diff
@@ -28,7 +28,10 @@
   a {
     color: #FF3464;
   }
-    </style>
+  .no-margin > div {
+    margin: 0px;
+  }
+  </style>
```

Comment:
Not ideal solution but selects the correct element to remove the 48px margin-top it was adding. I could not find a way to do this cleaner.